generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -54,7 +54,11 @@ import {
         Title,
         {
             provide: LOCALE_ID,
+        <%_ if (nativeLanguage === 'ar-ly') { _%>
+            useValue: 'en'
```

Comment:
ok, but I dont like hardcoding this for a language. Not sure if we will have same issue for other languages which might have non standard numbers. Anyway dont worry I'll change it later
@jhipster/developers any idea about other languages that might have similar issue here so that I can exclude them as well?